oggm/utils.py

```diff
@@ -2904,6 +2903,12 @@ def grid(self):
         """A ``salem.Grid`` handling the georeferencing of the local grid"""
         return salem.Grid.from_json(self.get_filepath('glacier_grid'))
 
+    @lazy_property
+    def rgi_area_km2(self):
+        """The glacier's RGI area (km2)."""
+        _area = gpd.read_file(self.get_filepath('outlines'))['Area'].values[0]
```

Comment:
Change to ``float(gpd.read_file(self.get_filepath('outlines'))['Area'].values[0])``

Reply:
or ``float(_area).round(decimals=3)`` at the line below, perhaps easier to read...